### ISSUE TITLE
Create ticket from staff interface

### ISSUE THREAD

**Issue by scramatte** [evolutionscript/HelpDeskZ-1.0#40]
It will be interesting to add  feature to create ticket from staff interface

**Comment by andysavage** [comment]
@scramatte   yes, that would be excellent, not only for ease of use but it would also make the interface less confusing. When you leave the admin interface it's far from clear how to get back to it. The less we need to leave it, the better!

I'm hoping this project gets more interest. I'm not a coder but I'd be willing to spend some time cleaning the spam out of the forum if that's any use?
Andy
